unime/src/routes/badges/[id]/+page.svelte

```diff
@@ -49,29 +49,18 @@
     isFavorite = credential.metadata.is_favorite;
     title = credential.metadata.display.name || credential.data.type.at(-1);
     icon = credential.metadata.display.icon || 'User';
-    // color =
-    //   credential.metadata.display.color ||
-    //   colors.at(
-    //     credential.id
-    //       .match(/[0-9]+/)
-    //       .at(0)
-    //       .at(0) % 8, // TODO: omits last value (white)
-    //   );
   }
 
-  // create entries to be shown
-  const { enrichment, ...entries } = credential.data.credentialSubject.achievement;
-  // entries['issuer'] = credential.data.issuer ?? credential.issuer_name;
-  // entries['issuanceDate'] = new Date(credential.data.issuanceDate).toLocaleString('en-US', {
-  //   dateStyle: 'long',
-  //   timeStyle: 'medium'
-  // });
+  const hiddenStandardFields: string[] = ['id', 'type', 'name', 'description', 'image'];
+  const hiddenCustomFields: string[] = ['enrichment'];
+
+  const entries = { ...credential.data.credentialSubject.achievement };
```

Comment:
I don't know if you want to modify the existing field, but if you don't maybe use structuredClone(...).

Reply:
Why? I'm not modifying anything, just unpacking through [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) and assigning it to a new const.